libs/core/src/lib/micro-process-flow/components/micro-process-flow/micro-process-flow.component.scss

```diff
@@ -0,0 +1,214 @@
+@import "../../mixins.scss";
+@import "../../variables.scss";
+
+// TODO: Move all styles to fundamental-styles library
+
+.#{$block} {
+    @include fd-reset();
+    @include fd-flex();
+
+    max-width: $max-container-width;
+    overflow: hidden;
+
+    &--independent-steps {
+        .#{$block}__connector {
+            &:before {
+                border-color: transparent;
+            }
+        }
+    }
+}
+
+.#{$block}__wrapper {
+    @include fd-reset();
+    @include fd-flex();
+
+    //transition: transform .3s ease;
+}
+
+.#{$block}__overflow-container {
+    @include fd-reset();
+    @include fd-flex();
+    width: 100%;
+    overflow: hidden;
+}
+
+.#{$block}__link-previous, .#{$block}__link-next {
+    @include fd-reset();
+    @include fd-flex();
```

Comment:
your `fd-flex` mixin should have @content and then group flex-related rules like:
```
@include fd-flex() {
    justify-content: flex-start;
    align-items: stretch;
}
```